### DISCUSSION TITLE
Issue template config and YAML file extensions

### DISCUSSION THREAD

**Discussion by michael-parkins** [community/community#4569] [Projects and Issues]
It appears that the config file that can be created in `.github/ISSUE_TEMPLATE` only works when it uses the `.yml` (i.e. `config.yml`) extension. I was not able to get `config.yaml` working in the same directory.

**Comment by cjbarth** [comment]
Duplicated by #13853

**Comment by askpatrickw** [comment]
Worse. The Template files CAN have .yaml extensions, but the config file cannot so the behavior is very inconsistent.


>NOTE: The YAML Guidelines say .yaml is the preferred file name extension. https://yaml.com/faq/
